modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/context.tsx

```diff
@@ -197,9 +200,28 @@ const viewReducer = (state: TState, action: TAction) => {
 				);
 			});
 
+			const newObjectViewColumns: TObjectViewColumn[] = [];
+
+			newObjectFields.map((objectField: TObjectField) => {
+				objectViewColumns.map((viewColumn: TObjectViewColumn) => {
+					if (objectField.name === viewColumn.objectFieldName) {
+						newObjectViewColumns.push({
+							...viewColumn,
+							label: objectField.label[defaultLanguageId],
+						});
+					}
+				});
+			});
```

Comment:
@danvb112 can we use forEach instead of a map?

Or return something from the map call and build the array from this return

Reply:
Yes @gabrielwas we can use it no problem